src/components/dashboard/menu.tsx

```diff
@@ -2,33 +2,65 @@
 import clsx from 'clsx'
 import Link from 'next/link'
 import { usePathname } from 'next/navigation'
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
-import { MenuItems } from "@/constants/menuItems";
-  
+import { MenuItems } from '@/constants/menuItems'
+import http from '@/configs/axiosConfig'
+import packageInfo from '../../../package.json'
+
 export default function Menu() {
   const pathname = usePathname()
   const { t } = useTranslation()
+
+  const [versionInfo, setVersionInfo] = useState('')
+  const UIVersion = packageInfo.version
+
+  useEffect(() => {
+    const fetchData = () => {
+      http
+        .get('/public/actuator/info')
+        .then((response) => {
+          const APIVersion = response.data.application.version
+          setVersionInfo(`UI v${UIVersion} | API v${APIVersion}`)
+        })
+        .catch((error) => {
+          console.error('Error fetching data:', error)
+        })
+    }
+
+    fetchData()
+  }, [])
+
   return (
-    <nav className="grid items-start gap-1 text-sm font-medium md:p-2 w-full">
-      {MenuItems.map((item) => {
-        const LinkIcon = item.icon
-        return (
-          <Link
-            key={item.key}
-            href={item.key}
-            className={clsx(
-              'flex items-center gap-3 rounded-lg px-3 py-2 text-muted-foreground transition-colors hover:text-primary',
-              {
-                'bg-slate-200 text-black dark:bg-slate-600/50 dark:text-slate-50 font-semibold': pathname === item.key,
-              },
-            )}
-          >
-            <LinkIcon className="h-6 w-6" />
-            {t(item.label)}
-          </Link>
-        )
-      })}
-    </nav>
+    <div className="flex flex-col h-full justify-between">
+      <nav className="grid items-start gap-1 text-sm font-medium md:p-2 w-full">
+        {MenuItems.map((item) => {
+          const LinkIcon = item.icon
+          return (
+            <Link
+              key={item.key}
+              href={item.key}
+              className={clsx(
+                'flex items-center gap-3 rounded-lg px-3 py-2 text-muted-foreground transition-colors hover:text-primary',
+                {
+                  'bg-slate-200 text-black dark:bg-slate-600/50 dark:text-slate-50 font-semibold':
+                    pathname === item.key,
+                },
+              )}
+            >
+              <LinkIcon className="h-6 w-6" />
+              {t(item.label)}
+            </Link>
+          )
+        })}
+      </nav>
+      {versionInfo && (
+        <div className="flex justify-center py-2">
+          <span className="text-xs text-gray-500 dark:text-gray-400">
+            {versionInfo}
```

Comment:
Versiyon infoyu yukarıda api içinde setlemek yerine burada setleyebilir misin ? Versiyon info şartı buraya eklemişsin ama teknik olarak bu her zaman true döner, çünkü versiyon BE'den dönmese bile diğer data ve içerdeki stringler versiyonInfo value'sunu truthy yapar. versionInfo değerine sadece BE'den gelen değeri verip yukarıda api içindeki tanımladığın değerleri burada tanımlarsan daha iyi olur. Böylece versionInfo şartı da çalışmış olur.

Reply:
versiyon BEden dönmediği zaman içinde başka string olmadığı için false dönüyor ve 2 versiyon da gözükmüyor. BEden versiyon alamadığımız zaman için sadece UI göstermek mi daha iyi olur yoksa hiç göstermemek mi? onun dışında değer tanımlamalarının yerini değiştirebilirim sorun değil

Reply:
API versiyon alamadığımız zaman sadece UI versiyon gösterecek, API versiyonu aldığımızda ikisini de gösterecek şekilde değiştirdim. göstermeyle alakalı şartı da kaldırdım.